SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -43,6 +52,7 @@ public static Crashes Instance
 
         public void OnChannelGroupReady(IChannelGroup channelGroup, string appSecret)
         {
+            PlatformSetEnabledAsync(true);
```

Comment:
Why do you call this one here?

Reply:
It is just a naive way to get crashes started, the PR to do the enable/disable logic will overwrite it.